src/app/globals.css

```
@@ -11,3 +7,154 @@
+
+  .dark .blog-post-markdown pre {
+    border-color: rgb(64 64 64);
+    box-shadow: 6px 6px 0 0 rgba(0, 229, 255, 0.35);
+  }
+
+  .blog-post-markdown pre code {
+    display: block;
+    min-width: 0;
+    white-space: pre-wrap;
+    overflow-wrap: anywhere;
+    word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)